src/base/bittorrent/session.cpp

```diff
@@ -90,6 +90,7 @@
 #include "base/utils/net.h"
 #include "base/utils/random.h"
 #include "base/version.h"
+#include "base/tagset.h"
```

Comment:
Isn't it already included? Maybe in `session.h`?
Anyway this place is incorrect (broken alphabetical order).